velox/exec/Spiller.h

```diff
@@ -37,8 +37,9 @@ class Spiller {
     kHashJoinProbe = 3,
     // Used for order by.
     kOrderBy = 4,
+    // Number of spiller types.
+    kNumTypes = 5,
```

Comment:
@duanmeng @xiaoxmeng Curious, whats the issue with previous code (except for Type : int8_t)? Previous code looks more readable.

Reply:
@amitkdutta previous code kNumTypes is 4 so some test can't cover. By putting kNumTypes into enum, we can kind of preventing this.